src/org/opendatakit/briefcase/ui/PushTransferPanel.java

```diff
@@ -348,7 +376,7 @@ private void setActiveTransferState(boolean active) {
       btnDestinationAction.setEnabled(true);
       btnSelectOrClearAllForms.setEnabled(true);
       btnTransfer.setEnabled(true);
-      // disable cancel button
+      btnDelete.setEnabled(formTransferTable.getSelectedForms().size() > 0);
```

Comment:
Just making sure you’re aware of `isEmpty` on `List`. It’s not important with small lists like this one, but it might be a good habit to use `isEmpty` because in some cases (`LinkedList`, for example), `size` is more expensive.

(Scala has a `nonEmpty` which is quite convenient, but `!isEmpty` is not so bad.)

Reply:
Of course I'm, shame on me :) It's fixed.